### PR TITLE
[AP-9631]: Added email as String search param 

### PR DESCRIPTION
**Pull request by jkiddo**

### CODE REVIEW

**Comment by jkiddo** [comment]
Change `email` to StringAndListParam

**Comment by ttntrifork** [comment]
Do we need to run `$mark-all-resources-for-reindexing` after deploying the IG version with this change?
In that case, we should probably bump the major or minor version of the IG before we merge to main, otherwise it will be auto-included in next build of eir-server and email search will stop working until someone runs `$mark-all-resources-for-reindexing` I assume.

We could bundle the execution of `$mark-all-resources-for-reindexing` with the data-migration job as that job runs after the new eir-server is deployed to prod anyway, so it is the correct time to run the `$mark-all-resources-for-reindexing` job

**Comment by jkiddo** [comment]
Ill check and get back

**Comment by jkiddo** [comment]
Execution of `$mark-all-resources-for-reindexing` IS needed after installing the IG

File: input/fsh/instances/searchparameters.fsh [hunk 1]
```diff
@@ -87,16 +87,19 @@ Usage: #definition /* ---- adding the #definition flag puts it in the package fo
 * component[=].expression = "Patient"
 * xpathUsage = #normal
 
-Instance: OrganizationEmail
+Instance: individual-extra-email
 InstanceOf: SearchParameter
-Description: "Extra search parameter enabling querying organizations for email contact point"
```
Comment (ttntrifork) [inline input/fsh/instances/searchparameters.fsh:92]: Could you write in Description why we create this "extra-email" parameter? To be able to search case-insensitive and using `:contains`

**Pull request merged**